api/room/models.py

```diff
@@ -39,6 +39,7 @@ class Room(Base, Utility):
     firebase_token = Column(String, nullable=True)
     cancellation_duration = Column(Integer, default=10)
     state = Column(Enum(StateType), default="active")
+    next_sync_token = Column(String, nullable=True)
```

Comment:
Hello @caveinn  and @dnuwa , Kindly throw more light on what the next_sync_token is for and how it is gotten. Thanks

Reply:
The next_sync_token is a token provided by google that allows you to get only the data that has changed since you last api request.